portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,120 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <head>
+        <meta charset="UTF-8">
+        <title>Sarah Abdulkerim | Portfolio</title>
+        
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB9bxbeKJgcnN8iVIWcrWTEITev5TbRQBI"></script>
+        <script src="script.js"></script>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+  
+    <body onload = "onLoadFuncs()"></body>
+
+    <div class="parallax">
+
+        <div class = "caption">
+            <img src = "images/profile.jpg"></img>
+            <br><br><br>
+            <span class = "border">
+                Sarah Abdulkerim.<br><br><br>Wellesley College | Computer Science
+            </span>
+            <h3><a href = '#intro'>Introduction</a></h3>
+        </div>
     </div>
-  </body>
+
+    <div class = "info" id = "intro">
+        <h3>
+            Welcome to my portfolio! <br>
+            I am Sarah Abdulkerim. <br>
+            I am a student studying Computer Science at Wellesley College.<br>
+            <a href = '#about'>About</a>
+        </h3>
+    </div>
+
+    <div class="parallax2"></div>
+
+    <div class = "info" id = "about">
+        <h3>About </h3>
+            <p> My name is Sarah Abdulkerim. I am a student studying Computer Science at Wellesley College.
```

Comment:
Maybe just "...I am studying Computer Science..." instead of "...I am a student studying Computer Science..."?